Vagrantfile

```diff
@@ -18,8 +18,8 @@ Vagrant.configure("2") do |config|
   config.vm.provider :virtualbox do |vb|
     vb.customize [
       "modifyvm", :id,
-      "--memory", "1024",
-      "--cpus", "2",
+      "--memory", "4096",
+      "--cpus", "4",
```

Comment:
Oh, that's right. I changed the memory and CPU for the boxes from 1GB and 2 to 4GB and 4, since they aren't in parallel right now (see #2). Is this a problem for anyone? (@mpapis?)